### DISCUSSION TITLE
How can I auto reload configuration when Basic Authentication was used?

### DISCUSSION THREAD

**Discussion by zhuqw12** [prometheus-operator/prometheus-operator#5407] [Q&A]
I tried to use prometheus-config-reloader. But it seems that prometheus-config-reloader does not support basic authentication. So I wonder is there a way to auto reload configuration when I use basic authentication ?

**Comment by simonpasquier** [comment]
related to https://github.com/prometheus-operator/prometheus-operator/issues/4200?

**Reply by zhuqw12** [reply]
I checked this, thanos could do that. 
Thanks.
